scripts/create_static_narrative.py

```diff
@@ -0,0 +1,203 @@
+"""Script to create a static narrative from the commandline."""
+
+import argparse
+import logging
+import sys
+from configparser import ConfigParser
+from os import environ
+from typing import Any
+
+from lib.installed_clients.WorkspaceClient import Workspace
+from lib.StaticNarrative.creator import StaticNarrativeCreator
+from lib.StaticNarrative.narrative_ref import NarrativeRef
+
+DEPLOY = "KB_DEPLOYMENT_CONFIG"
+SERVICE = "KB_SERVICE_NAME"
+AUTH = "auth-service-url"
+
+
+def get_config_file() -> str:
+    """Get the path to the config file.
+
+    :return: path to the config file
+    :rtype: str
+    """
+    return environ.get(DEPLOY, "./test/deploy.cfg")
+
+
+def get_service_name() -> str:
+    """Get the service name.
+
+    :return: service name
+    :rtype: str
+    """
+    return environ.get(SERVICE, "StaticNarrative")
+
+
+def get_config() -> None | dict[str, str]:
+    """Retrieve the config from the config file.
+
+    :return: parsed config as a dictionary
+    :rtype: None | dict[str, str]
+    """
+    if not get_config_file():
+        return None
+    retconfig = {}
+    config = ConfigParser()
+    config.read(get_config_file())
+    for nameval in config.items(get_service_name() or "StaticNarrative"):
```

Comment:
Could also do `for name, value in config.items()` I'm not sure which is more readable. Just a thought.